docs/faq.md

```diff
@@ -40,3 +40,316 @@ endpoints, causing the verification to fail, and then the pod operation will fai
+- **Workload resources**: Deployments, StatefulSets, DaemonSets
+- **Custom resources**: All Kruise CRDs
+- **Webhook management**: ValidatingWebhookConfiguration, MutatingWebhookConfiguration
+- **Node access**: For kruise-daemon operations
+
+The Helm chart automatically creates appropriate RBAC resources. For custom installations, refer to the RBAC manifests in the charts.
+
+#### **Q**: How do I secure OpenKruise in production?
+
+**A**: Follow these security best practices:
+- **Network policies**: Restrict kruise-system namespace traffic
+- **Pod security**: Use Pod Security Standards/Policies
+- **RBAC**: Apply principle of least privilege
+- **TLS certificates**: Use cert-manager for webhook certificates
+- **Image security**: Scan OpenKruise images for vulnerabilities
+- **Audit logging**: Enable Kubernetes audit logs for Kruise operations
+
+```yaml
+# Example network policy
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: kruise-system-policy
+  namespace: kruise-system
+spec:
+  podSelector: {}
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          name: kube-system
+```
+
+#### **Q**: Can I run OpenKruise in air-gapped environments?
+
+**A**: Yes, OpenKruise supports air-gapped deployments:
+1. **Mirror images**: Copy OpenKruise images to private registry
+2. **Update image references**: Modify Helm values to use private registry
+3. **Bundle charts**: Download Helm charts for offline installation
+4. **Configure image pull secrets**: Set up authentication for private registry
+
+```bash
+# Example air-gapped installation
+helm install kruise ./kruise-chart \
+  --set manager.image.repository=private-registry.com/kruise-manager \
+  --set daemon.image.repository=private-registry.com/kruise-daemon \
+  --set imagePullSecrets[0].name=private-registry-secret
+```
+
+## Monitoring and Observability
+
+#### **Q**: How do I monitor OpenKruise workloads?
+
+**A**: OpenKruise provides comprehensive monitoring capabilities:
+- **Built-in metrics**: Prometheus metrics on port 8080
+- **Kruise State Metrics**: Extended metrics for workload states
+- **Structured logging**: JSON format logs for better parsing
+- **Custom dashboards**: Grafana dashboards for visualization
+
+```bash
+# Install kruise-state-metrics
+helm install kruise-state-metrics openkruise/kruise-state-metrics
+
+# Access metrics
+kubectl port-forward -n kruise-system svc/kruise-controller-manager 8080:8080
+curl localhost:8080/metrics
+```
+
+#### **Q**: What metrics should I monitor for OpenKruise?
+
+**A**: Key metrics to monitor:
+- **Controller performance**: `controller_runtime_reconcile_time_seconds`
+- **Workqueue depth**: `workqueue_depth`
+- **Webhook latency**: `controller_runtime_webhook_latency_seconds`
+- **Error rates**: `controller_runtime_reconcile_errors_total`
+- **Resource states**: Custom metrics from kruise-state-metrics
```

Comment:
plz add a link to the [trouble-shooting page](https://openkruise.io/docs/operator-manuals/troubleshooting) of the operation manual